.github/workflows/build.yaml

```
@@ -55,3 +61,70 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  wheel-build-pylibcugraph:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@upkeep/manylinux-improvements
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cugraph
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: pylibcugraph
+      package-dir: python/pylibcugraph
+
+      # Note that this approach to cloning repos obviates any modification to
+      # the CMake variables in get_cumlprims_mg.cmake since CMake will just use
+      # the clone as is.
+      extra-repo: rapidsai/cugraph-ops
+      extra-repo-sha: branch-22.12
+      extra-repo-deploy-key: CUGRAPH_OPS_SSH_PRIVATE_DEPLOY_KEY
+
+      versioneer-override: ${{ inputs.wheel_versioneer_override }}
+      skbuild-configure-options: "-DDETECT_CONDA_ENV=OFF -DCUGRAPH_BUILD_WHEELS=ON -DCPM_cugraph-ops_SOURCE=/project/cugraph-ops/"
+  wheel-publish-pylibcugraph:
+    needs: wheel-build-pylibcugraph
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-publish.yml@upkeep/manylinux-improvements
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cugraph
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: pylibcugraph
+      build-tag: ${{ inputs.wheel_build_tag }}
+  wheel-build-cugraph:
+    needs: wheel-publish-pylibcugraph
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@upkeep/manylinux-improvements
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cugraph
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: cugraph
+      package-dir: python/cugraph
+      versioneer-override: ${{ inputs.wheel_versioneer_override }}
+
+      # Note that this approach to cloning repos obviates any modification to
+      # the CMake variables in get_cumlprims_mg.cmake since CMake will just use
+      # the clone as is.
+      extra-repo: rapidsai/cugraph-ops
+      extra-repo-sha: branch-22.12
```

As with cumlprims I think we we could probably use inputs.sha here instead, up to you.
```suggestion
      extra-repo-sha: branch-23.02
```